ci/cloudbuild/builds/check-api.sh

```diff
@@ -114,6 +114,12 @@ function check_abi() {
       # by some other number indicating the length of the symbol within the
       # "internal" namespace. See: https://en.wikipedia.org/wiki/Name_mangling
       -skip-internal-symbols "(8internal|_internal)\d"
+      # We also ignore the raw gRPC Stub class. The generated gRPC headers that
+      # contain these classes are installed alongside our headers. When a new
+      # RPC is added to a service, these classes gain a pure virtual method. Our
+      # customers do not use these classes directly, so this should not
+      # constitute a breaking change.
+      -skip-internal-types "::StubInterface"
```

Comment:
This is an unmangled symbol, but based on the previous comment we should be skipping mangled symbols.  Which one is the "Right Thing"[tm]?

Reply:
I think either will work.
https://github.com/lvc/abi-compliance-checker/blob/8caa4417e5ccbd1cbf6add727842e205d6568cb4/abi-compliance-checker.pl#L620-L626

`-skip-internal-types` is easier to read. The reason I did not move the `"(8internal|_internal)\d"` business into types, as `"(::internal::|_internal::)"` is because I did not feel like testing it 😬

Reply:
For future record, this is from the logs linked in the PR description: 
![image](https://user-images.githubusercontent.com/23088558/205461995-62a45325-ad14-4a12-99f5-4438ffb2220b.png)